### PR TITLE
refactor: postgres username and password stored in keyvault (CMC-1494)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1494

### Change description ###

Store postgres credentials in keyvault

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -29,7 +29,9 @@ def secrets = [
   'unspec-${env}': [
     secret('microservicekey-unspec-service', 'S2S_SECRET'),
     secret('ccd-importer-username', 'CCD_CONFIGURER_IMPORTER_USERNAME'),
-    secret('ccd-importer-password', 'CCD_CONFIGURER_IMPORTER_PASSWORD')
+    secret('ccd-importer-password', 'CCD_CONFIGURER_IMPORTER_PASSWORD'),
+    secret('postgres-user', 'POSTGRES_USER'),
+    secret('postgres-password', 'POSTGRES_PASSWORD')
```
Comment (dharmendrak) [inline Jenkinsfile_CNP:35]: Hey Sam, How we are adding these vault values in unspec vaults, manually or copying from another camunda vault?

Comment (swalker125) [reply Jenkinsfile_CNP:35]: manually

File: charts/civil-damages-service/values.preview.template.yaml [hunk 2]
```diff
@@ -45,12 +45,14 @@ java:
         - robotics-notification-sender
         - robotics-notification-recipient
         - launch-darkly-sdk-key
+        - postgres-user
+        - postgres-password
```
Comment (pd-dudkiewicz) [inline charts/civil-damages-service/values.preview.template.yaml]: we don't have to pull it here, it's not used in our code

**Review by dharmendrak — DISMISSED** [review]

**Comment by pd-dudkiewicz** [comment]
@swalker125 did you add new secrets to demo vault as well? if not, then the build will fail there trying to pull them

File: charts/civil-damages-service/values.preview.template.yaml (hunk 2 above)
Comment (swalker125) [reply charts/civil-damages-service/values.preview.template.yaml]: hangover from playing around, good spot

**Review by pd-dudkiewicz — APPROVED** [review]

**Pull request merged**